js/views/modals/listingDetail/Listing.js

```diff
@@ -508,6 +549,11 @@ export default class extends BaseModal {
       this.setSelectedPhoto(this.activePhotoIndex);
       this.setActivePhotoThumbnail(this.activePhotoIndex);
       this.adjustPriceBySku();
+      this.$reviews = this.$('.js-reviews');
+
+      // get the ratings data, if any
```

Comment:
Not sure if you want to fight this battle... I've logged many server issue today, so I'm over it, but ideally the server would return a 200 and an empty array if there are no reviews instead of:

```
{
    "success": false,
    "reason": "no link named 'index.json' under QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn"
}
```

As it is now, there's really no way to distinguish between a user having no reviews and there being an error fetching them. We may be able to punt on this if we decide to both silently ignore errors and no reviews.

(but as soon as Wolf asks for a 'This listing has no reviews' placeholder punting is not an option)